### PR TITLE
Fix Search Results navigation for #762

### PR DESCRIPTION
**Pull request by RedLeader16**
# Description
This MR fixes the bug where navigation results do not route the user to the resource. 
It also addresses a bug in the breadcrumbs where the query params display.

**Working Search**

https://user-images.githubusercontent.com/34227093/234102309-17c5a4eb-8f4b-491c-b954-e7d43999a2d0.mov

**BreadCrumbs fix:**
before:
beforeFix

after:
afterFix

### Does this correspond to an open issue?

https://github.com/featureform/featureform/issues/762

Unit Tests: 
image

### Select type(s) of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have fixed any merge conflicts

### CODE REVIEW

File: dashboard/src/components/searchresults/SearchResultsView.js [hunk 1]
```diff
@@ -112,42 +113,39 @@ const SearchResultsList = ({ type, contents, setVariant }) => {
...
+  "LABEL_VARIANT": "Label",
+  "TRAINING_SET": "TrainingSet",
+  "TRAINING_SET_VARIANT": "TrainingSet",
+  "SOURCE": "Source",
+  "SOURCE_VARIANT": "Source",
+  "PROVIDER": "Provider",
+  "ENTITY": "Entity",
+  "MODEL": "Model",
+  "USER": "User",
 }
```
Comment (RedLeader16) [inline dashboard/src/components/searchresults/SearchResultsView.js:128]: @sdreyer for the purposes of search, do variant mappings point back to the original type?

Comment (simba-git) [reply dashboard/src/components/searchresults/SearchResultsView.js:128]: Not sure I understand the question but you also have to think about the situation where I'm searching for a specific variant. we also don't want to clog up a search result with all of the variants if I'm searching for a feature.

Ex.
If I search for "abc"
And I have a feature variant with name F1 and Variants [abc, def, ghi], then the abc variant should show up in the results.

**Review by simba-git — APPROVED** [review]
Great work

File: dashboard/src/components/searchresults/SearchResultsView.js (hunk 1 above)
Comment (RedLeader16) [reply dashboard/src/components/searchresults/SearchResultsView.js:128]: Gotcha, that makes sense to me. After the user clicks a variant search result, the existing code has a "hasVariant" flag that handles the routing. All clear!

**Pull request merged**